src/containers/pages/IRS/plans/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -159,6 +159,246 @@ exports[`components/IRS Reports/IRSPlansList renders plan definition list correc
 </h3>
 `;
 
-exports[`components/IRS Reports/IRSPlansList renders plan definition list correctly: table headers 1`] = `null`;
+exports[`components/IRS Reports/IRSPlansList renders plan definition list correctly: table headers 1`] = `
+<div
```

Comment:
Questions:

- why is it changing from null to something else? (looks like previously the null was intentional)
- Does this snapshot need to be this large just to show that headers were rendered?

Reply:
I think it should not have been `null` if you look at the [test](https://github.com/onaio/reveal-frontend/blob/87dc5b5186a5f9e059377fe134d0adf9a5c9a0f2/src/containers/pages/IRS/plans/tests/index.test.tsx#L77), at that point table headers should be rendered.